Gemfile.lock

```diff
@@ -2,36 +2,36 @@ GEM
   remote: http://rubygems.org/
```

Comment:
It looks like we have the important dependencies locked down to a minor version ... so I guess this is safe to do.